.gitignore

```diff
@@ -75,3 +75,5 @@ Videos/*
 !Videos/test.mp4
 slowmovie.log
 slowmovie.conf
+# python venv
+.venv
```

Comment:
should this be `.SlowMovie` now?

Reply:
Yep, I overlooked changing it there. I just pushed a correction.